tests/conftest.py

```diff
@@ -77,23 +79,49 @@ def example_atoms(num_data: int, pbc: bool, calc_results: List[str]) -> Atoms:
     return atoms_list
 
 
+@pytest.fixture()
+def example_dataset(num_data: int) -> Atoms:
+    atoms_list = []
+
+    p2 = np.random.uniform(low=1.0, high=1.5, size=(num_data,))
+    for i in range(num_data):
+        positions = np.array([[0.0, 0.0, 0.0], [0.0, 0.0, p2[i]]])
+
+        additional_data = {}
+        additional_data["cell"] = [0.0, 0.0, 0.0]
+
+        atoms = Atoms("H2", positions=positions, **additional_data)
+        atoms.calc = EMT()
+        atoms.get_potential_energy()
+        atoms.get_forces()
+
+        atoms_list.append(atoms)
+
+    return atoms_list
+
+
 @pytest.fixture()
 def get_tmp_path(tmp_path_factory):
     test_path = tmp_path_factory.mktemp("apax_tests")
     return test_path
 
 
 @pytest.fixture(scope="session")
-def get_md22_stachyose(get_tmp_path):
+def tmp_data_path(tmp_path_factory):
+    test_path = tmp_path_factory.mktemp("data")
+    return test_path
+
+
+@pytest.fixture(scope="session")
+def get_md22_stachyose(tmp_data_path):
     url = "http://www.quantum-machine.org/gdml/repo/static/md22_stachyose.zip"
-    data_path = get_tmp_path / "data"
-    file_path = data_path / "md22_stachyose.zip"
+    file_path = tmp_data_path / "md22_stachyose.zip"
 
-    os.makedirs(data_path, exist_ok=True)
+    os.makedirs(tmp_data_path, exist_ok=True)
     urllib.request.urlretrieve(url, file_path)
 
     with zipfile.ZipFile(file_path, "r") as zip_ref:
-        zip_ref.extractall(data_path)
+        zip_ref.extractall(tmp_data_path)
```

Comment:
the goal of this test in particular is to see whether we break anything subtle about the model that changes the quality of models we train.

But this would of course be a great way to test things that we need trained models for. While it might be overkill to add a zntrack dependency for that to apax directly, perhaps we could set up a separate repository for these purposes.

Reply:
It would just be a test dependency

Reply:
Oh that's true! Good suggestion, I'll create an issue.